### ISSUE TITLE
Provide support for yarn berry & pnpm

### ISSUE THREAD

**Issue by ibraelillo** [serverless-stack/open-next#2]
Hi, can you provide support for pnpm and yarn berry package managers?  

> @astore/app@0.0.0 predeploy /home/ibra/accor/astore/packages/app
> open-next build

>node:internal/errors:477
>    ErrorCaptureStackTrace(err);
>    ^
>
>Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@vercel/next' imported from /home/ibra/accor/astore/node_modules/.pnpm/open-next@0.0.1/node_modules/open-next/dist/build.js
    at new NodeError (node:internal/errors:387:5)
    at packageResolve (node:internal/modules/esm/resolve:951:9)
    at moduleResolve (node:internal/modules/esm/resolve:1000:20)
    at defaultResolve (node:internal/modules/esm/resolve:1214:11)
    at nextResolve (node:internal/modules/esm/loader:165:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:844:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:431:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

**Comment by fwang** [comment]
Fixed in v0.1.0.

^ make sure to run `open-next` with the `latest` npm version. ie. `npx open-next@latest build`

**Comment by trustyoo86** [comment]
@fwang It doesn't seem to be a problem with the @vercel/next package.
Yarn berry can be resolved @vercel/next package error with packageExtensions in `.yarnrc.yml`
But, In my case, I'm getting error in post install of @vercel/build-utils package.

@vercel/build-utils package uses node_modules, but it seems that the module cannot be read in yarn berry's pnp mode.
Is there any other issue solutions?

Here is my build error log

```bash
# This file contains the result of Yarn building a package (esbuild@npm:0.15.18)
# Script name: postinstall

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/Users/x/open-next-test-app/.pnp.cjs'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at Module._preloadModules (node:internal/modules/cjs/loader:1282:12)
    at loadPreloadModules (node:internal/bootstrap/pre_execution:539:5)
    at prepareMainThreadExecution (node:internal/bootstrap/pre_execution:85:3)
    at node:internal/main/run_main_module:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
```
